.github/actions/veracode/source_clear.sh

```diff
@@ -4,10 +4,7 @@ echo "=========================== Starting SourceClear Script ==================
 PS4="\[\e[35m\]+ \[\e[m\]"
 set +e -v -x
 
-mvn -B -q clean install \
```

Comment:
How does `SRCCLR_API_TOKEN` get injected now? Does the new `srcclr` command just expect it as an env var?

Reply:
https://docs.veracode.com/r/Veracode_SCA_Agent_Environment_Variables

Reply:
Yep, it uses env variables or stuff it can retrieve from github repository on its own, I'm only overriding the URI there to use the automatic project creation mechanism.